session_state.go

```diff
@@ -75,20 +76,39 @@ func (sm *stateMachine) Disconnected(session *session) {
 	}
 }
 
+var msgTypeExecutionReport = []byte("\x0135=8\x01")
+
+func IsExecutionReport(m []byte) bool {
+	return bytes.Contains(m, msgTypeExecutionReport)
```

Comment:
35=8 should be the first field I believe but not 100% - if so, can also use `bytes.HasPrefix` instead

Reply:
seems like it should be a rule: https://github.com/alpacahq/quickfix/blob/65ca3790cc5ce48ebdff2fae2b33d9689dd627cf/message.go#L184